index.qmd

```diff
@@ -39,73 +40,58 @@ format:
           margin-left: 0
         embed-resources: true
         page-layout: full
-my-custom-stuff:
-   my-reuseable-variable: "I can use this wherever I want in the markdown, and change it in only once place :)"
 ---
 
-## Contents
 
-Some example slides - [also look at example RevealJS slides in the Quarto docs](https://quarto.org/docs/presentations/revealjs/)
+## Welcome
 
-* Non-executable and executable code-blocks
-* bullet points with highlighting 
-* two-column slides 
-* how to include a slide from a separate MD file
-* preview and link to a webpage
+![](img/sainsbury-wellcome-centre.jpg)
 
-## Just a code block, nothing gets executed...
+## Neuroinformatics Unit
 
-... but there is some fancy highlighting
+![](img/niu-team.png)
 
-```{.python code-line-numbers="1|3|4-9"}
-from pathlib import Path
+## Code of Conduct
 
-home_path = Path.home()
-if home_path.exists():
-  data_path = home_path / "data"
-else:
-  pass
-  # raise some error maybe?
-```
+* Reminder: please follow the [code of conduct](https://docs.google.com/document/d/1pQGML9twa-GnqQNDtNRg1P2_Wwzvl1b61A6fLzpY_2Q/edit?tab=t.6jlq0oph6gve)
+* Summary: be kind, respectful and professional
+* Contacts: Adam, Sofia, Niko or Alessandro
 
-## A code block that's actually executed at render-time
+## Schedule
 
-```{python}
-#| echo: true
-#| code-fold: true
+![](img/schedule-2025.png)
 
-from pathlib import Path
+## Building map
 
-print("Hello world")
-```
+![](img/swc-maps.png)
```

Comment:
I would keep this map as is on the left hand side, and add the fire assembly map (the one at the bottom of our google doc) on the right using columns.